lib/rubocop/cop/style/indent_heredoc.rb

```diff
@@ -0,0 +1,127 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cops checks the indentation of the here document bodies. The bodies
+      # are indented one step.
+      # In Ruby 2.3 or newer, squiggly heredocs (`<<~`) should be used. If you
+      # use the older rubies, you should introduce some library to your project
+      # (e.g. ActiveSupport, Powerpack or Unindent).
+      #
+      # @example
+      #
+      #   # bad
+      #   <<-END
+      #   something
+      #   END
+      #
+      #   # good
+      #   <<~END
+      #     something
+      #   END
+      #
+      #   # good
+      #   <<~END.strip_heredoc
+      #     something
+      #   END
+      class IndentHeredoc < Cop
+        include ConfigurableEnforcedStyle
+
+        MSG = 'Use %d spaces for indentation in a heredoc, ' \
+              'relative to the start of the line where the ' \
+              'heredoc delimiter is.'.freeze
```

Comment:
Should the message be improved with the specific actions the user need to take? I'm thinking of the ruby 2.2 case, where auto-correct with the default configuration won't work.